docs/SRS/SRS.tex

```diff
@@ -365,24 +370,186 @@ \section{Project Issues}
+		\cline{2-3}    & Final Demonstration                & March 20 \textemdash{} 31, 2023   \\
+		\cline{2-3}    & Final Documentation                & April 5, 2023                     \\
+		\hline
+	\end{tabular}
+
+	\label{project_tasks}
+\end{table}
+
+\subsubsection{Planning of the Development Phases}
+
+The development of the project will be conducted in two phases:
+
+\begin{enumerate}
+	\item Initial development of application and documentation
+	\item Refinement of application and documentation
+\end{enumerate}
+
+Phase 1 is where the bulk of the application will be designed and implemented. The design of both
+the components of the application and how the components will interact will de developed.
+Additional, Phase 1 is where most of the documentation and report will be written. Phase 1 will end
+with the Revision 0 Demonstration. Here the stakeholders will see the application implementation
+and be able to provide feedback.
+
+Phase 2 will be focused on refining the application and the documentation. There is expected to be
+no new major feature development and instead, all efforts will be focused on incorporating the
+stakeholders' feedback into the application.
 
 \subsection{Migration to the New Product}
+\subsubsection{Requirements for Migration to the New Product}
+
+There are no requirements for migrating to the new product.
+
+\subsubsection{Data That Has to Be Modified or Translated for the New System}
+
+No data needs to be modified or translated to the new system.
 
 \subsection{Risks}
 
+\begin{itemize}
+	\item Failures in the work orders and quotes work flow may lead to data loss.
+	\item Failures in the appointments work flow may lead to a loss in appointment or conflicting
+	      appointments.
+	\item Failure to meet deadlines will cause set backs in project's timeline. In the event of this, lower
+	      priority requirements may need to be dropped.
+\end{itemize}
+
 \subsection{Costs}
 
+There are no financial costs associated with the development of this application. All software and
+cloud infrastructure used are free to use. There will be about six months of development time
+required.
+
 \subsection{User Documentation and Training}
+\subsubsection{User Documentation Requirements}
+
+The application will feature a ``Getting Started'' guide, where it shall guide the user through the
+most common use cases. For vehicle owners, the use cases will include: searching for shops,
+requesting quotes, and scheduling appointments. For automotive shops, the use cases include:
+setting shop details, managing appointments, managing employees, responding to quotes, and managing
+work orders.
+
+\subsubsection{Training Requirements}
+
+Knowledge of how to navigate a website will be required. Documentation concerning detailed usage of
+the website's user flows will be provided to the user.
 
 \subsection{Waiting Room}
 
+There are currently no requirements that are not part of the initial release.
```

Comment:
> The waiting room holds requirements that will not, for one reason or another, be part of the initial release of the product. If you are competent at gathering requirements, your users may often be inspired to think of more requirements than you can fit within the constraints of the project. While you may not want to include all of these requirements in the initial version of the product, neither do you want to lose them. 
> If you are doing iterative development then the waiting room is your backlog.

@TimChoy @LeonSo7 @joyxiao99
This could be a place where we add the requirements for the stretch goals. Thought?

Reply:
That sounds good to me. I'll leave the decision up to @joyxiao99 and @TimChoy